tests/foreman/ui/test_computeresource_azurerm.py

```diff
@@ -36,13 +36,13 @@
 
 
 @pytest.fixture(scope='module')
-def module_org(module_azurerm_cr):
-    return module_azurerm_cr.organization[0].read()
+def module_org():
+    return entities.Organization().create()
 
 
 @pytest.fixture(scope='module')
-def module_loc(module_azurerm_cr):
-    return module_azurerm_cr.location[0].read()
+def module_location(module_org):
```

Comment:
As we have already had the module scope fixture for this at https://github.com/SatelliteQE/robottelo/blob/43b38ade8c5f293c20afa43b120d178236767e19/pytest_fixtures/api_fixtures.py#L41 I would think we do not need this now. This is unblocking though.

Reply:
Good catch !!  this one not needed and removed in latest push.